libs/src/utils/network.test.js

```diff
@@ -35,10 +36,33 @@ describe('timeRequestsAndSortByVersion', () => {
     assert.strictEqual(res[4].request.url, 'https://error.audius.co')
   })
 
+  it('sortByVersion = false', async () => {
+    const requests = [
+      setupRequest('https://fastest.audius.co', 50, '1.2.3'),
```

Comment:
could be nice to have a node or two running ahead to represent case when version >= current version

Reply:
agreed, added